src/Role/Repository/RoleHierarchyRepository.php

```diff
@@ -37,9 +37,27 @@ private function getChildIds(array $parentIds): ?array
         $searchCriteria = $this->getDataObjectManager()
             ->whereIn('parent_role_id', $parentIds);
 
+        if($searchCriteria->count() == 0) return [];
```

Comment:
we dont use singleline ifs, please write it out to the fully

if() {
return stuff happens here
}